dev-utils/webdriver.js

```diff
@@ -345,6 +345,14 @@ function getBrowserInfo() {
   }
 }
 
+function getBrowserFeatures() {
```

Comment:
Not sure though, I prefer having 'browser' in the name since this is suppose to run in the context of the browser. Also, this returns an object containing for the features but those features might be supported or not (depending on the value)